File: Lab7/t4.java
import java.util.Scanner;
public class t4{
    public static void main(String[] args){
        Scanner scan = new Scanner(System.in);
        int[] matrix = new int[4];
        System.out.println("Please input matrix 1 (2*2): ");
        for (int i = 0; i < 4; i++)
            matrix[i] = scan.nextInt();
        System.out.println("Please input matrix 2 (2*2): ");
        for (int i = 0; i < 4; i++)
            matrix[i] += scan.nextInt();
        System.out.println("The SUM matrix is: ");
        System.out.printf("%d\t%d\n%d\t%d\n", matrix[0], matrix[1], matrix[2], matrix[3]);
    }
}

File: Lab7/t4.cpp
#include <stdio.h>
int main(){
    int matrix[4], tmp;//tmp to receive number in the second array
    puts("Please input matrix 1 (2*2): ");
    for (int i = 0; i < 4; i++)//scan number in first array
        scanf("%d", &matrix[i]);
    puts("Please input matrix 2 (2*2): ");
    for (int i = 0; i < 4; i++){//scan number in second array and add to first arr
        scanf("%d", &tmp);
        matrix[i] += tmp;
    }
    puts("The SUM matrix is: ");
    printf("%d\t%d\n%d\t%d\n", matrix[0], matrix[1], matrix[2], matrix[3]);
    return 0;
}